log/logger_setup.go

```diff
@@ -76,8 +92,11 @@ func configureSubLogger(logger, levels string, output io.Writer) error {
 // SetupSubLoggers configure all sub loggers with provided configuration values
 func SetupSubLoggers(s []SubLoggerConfig) {
 	for x := range s {
-		output := getWriters(&s[x])
-		err := configureSubLogger(strings.ToUpper(s[x].Name), s[x].Level, output)
+		output, err := getWriters(&s[x])
+		if err != nil {
+			return
```

Comment:
```suggestion
			fmt.Printf("Logger setup error unable to setup sub logger %v %v\n", s[x].Name, err)
			return
```
Normally I'd say this style error is pretty verbose, but its at setup and hopefully would let a user know they have a terrible config